components/profile/activityTab/OfferRow.vue

```diff
@@ -139,10 +144,15 @@
       <div v-else>
         {{ blank }}
       </div>
-      <div v-if="offer.expirationDate">
-        <span> {{ format(offer.expirationDate, EXPIRATION_FORMAT) }}</span>
-        <span class="text-k-grey ml-3"> ({{ formatToNow(offer.expirationDate, isExpired) }})</span>
-      </div>
+      <template v-if="offer.expirationDate">
+        <div v-if="ieExpired">
```

Comment:
typo here?

Reply:
nice catch, don't know how this one slipped through